### DISCUSSION TITLE
List.random* functions not available

### DISCUSSION THREAD

**Discussion by nkosi23** [dotnet/fsharp#17747] [Q&A] [answered]
For some reason, I am unable to access the List.random* functions of the List module [documented on this page](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-collections-listmodule.html#randomChoice). 

The functions simply do not exist, even though I can see that they have been committed two years ago, and they are present in the documentation anyway.

My project is targetting .NET 8.0 ans is using FSharp.Core 8.0.100 (as per the command `dotnet list package`)

```fsproj
<Project Sdk="Microsoft.NET.Sdk">

    <PropertyGroup>
        <OutputType>Exe</OutputType>
        <TargetFramework>net8.0</TargetFramework>
        <RootNamespace>ConsoleApp1</RootNamespace>
    </PropertyGroup>

    <ItemGroup>
        <Compile Include="Program.fs"/>
    </ItemGroup>

    <ItemGroup>
        <PackageReference Include="DnsClient" Version="1.8.0"/>
        <PackageReference Include="FSharp.Data" Version="6.4.0"/>
        <PackageReference Include="FSharp.SystemTextJson" Version="1.3.13"/>
    </ItemGroup>

</Project>

```

And i have opened the namespace FSharp.Collections using `open FSharp.Collections`

**Comment by nkosi23** [comment]
The functions were actually introduced in .NET 8.0.400, i didn't expect it.

**Reply by vzarytovskii** [reply, marked as answer]
Yes, they were merged around this summer, hence only present in 8.0.400
